Cargo.toml

```
@@ -5,7 +5,7 @@
 # PyPI. Installing this with `pip install blake3` is better than something
 # redundant like `pip install PyBLAKE3-py`.
 name = "blake3"
-version = "1.0.2"
+version = "1.0.3"
```

damn, not the first time I forgot this. will push 1.0.4
```suggestion
version = "1.0.4"
```

---

It's too bad that Cargo doesn't support dynamic versions similar to Python's `setuptools-scm` hook. https://github.com/rust-lang/cargo/issues/6583 has some suggestions, e.g. a `build.rs` hack. Maybe you could depend on `setuptools-scm` and shell out to `setuptools-scm` in `build.rs`? The Python tool gets its version from SCM and falls back to `PKG-INFO` file for sdists.